app/react/V2/Routes/Settings/Preserve/Preserve.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { FormEvent, useEffect, useState } from 'react';
```

Comment:
We should have a unit test for this, or at least create a task issue to have it.